plugins/Goals/lang/en.json

```
@@ -3,63 +3,63 @@
         "ColumnVisits": "The total number of visits, regardless of whether a goal was triggered or not.",
-        "ColumnVisitsProductDocumentation": "The number of visits on the Product\/Category page. This is also used to process the %s conversion rate. This metric is in the report if Ecommerce view tracking was setup on Product\/Category pages.",
+        "ColumnVisitsProductDocumentation": "The number of visits on the Product\/Category page. Also used to process the %s conversion rate. This metric is in the report if e-commerce view-tracking was set up on the Product\/Category pages.",
         "Contains": "contains %s",
-        "ConversionByTypeReportDocumentation": "This report provides detailed information about the goal performance (conversions, conversion rates and revenue per visit) for each of the categories available in the left panel. %1$s Please click on one of the categories to view the report. %2$s For more information, read the %3$sTracking Goals documentation%4$s",
+        "ConversionByTypeReportDocumentation": "This report provides detailed info about the goal performance (conversions, conversion rates and revenue per visit) for each of the categories available in the left panel. %1$s Please click on one of the categories to view the report. %2$s For more info, read the %3$sTracking goals documentation%4$s",
         "ConversionRate": "%s conversion rate",
         "ConversionRatePageViewedBefore": "Viewed before %s rate",
         "Conversions": "%s conversions",
         "ConversionsDescription": "conversions",
-        "ConversionsOverview": "Conversions Overview",
-        "ConversionsOverviewBy": "Conversions overview by type of visit",
+        "ConversionsOverview": "Conversion Overview",
+        "ConversionsOverviewBy": "Conversion overview by type of visit",
         "DaysToConv": "Days to Conversion",
-        "DaysToConvReportDocumentation": "This report shows how many days pass before your visitors convert a goal.",
+        "DaysToConvReportDocumentation": "This report shows how many days pass on average before your visitors convert a goal.",
         "Details": "Goal details",
         "DefaultGoalConvertedOncePerVisit": "(default) Goal can only be converted once per visit",
-        "DefaultRevenueLabel": "Goal default revenue",
-        "DefaultRevenueHelp": "For example, a Contact Form submitted by a visitor may be worth $10 on average. Matomo will help you understand how well your visitors segments are performing.",
-        "DeleteGoalConfirm": "Are you sure you want to delete the Goal %s?",
+        "DefaultRevenueLabel": "Default goal revenue",
+        "DefaultRevenueHelp": "For example, a contact form submitted by a visitor may be worth $10 on average. Matomo will help you understand how well your visitor segments are performing.",
+        "DeleteGoalConfirm": "Delete the %s goal?",
```

This is shown in a confirmation overlay. So I can't see a good reason for making the text shorter. Also the placeholder is replaced with the goals name, so it should be placed after `goal` I guess.

```suggestion
        "DeleteGoalConfirm": "Are you sure you want to delete the goal %s?",
```

---

With "%s goal" it is possible to get away without quoting it.
If it is "goal e-commerce", it needs to be "goal "e-commerce"".
There is no "Are you sure anywhere", since it wastes time reading it, and it has also fallen out of fashion.
The reasoning there is that it appears to be nagging.

---

In that case it might better to use quotes around the goal name. Goal names are actually user input and may contain any text. So not quoting them could break the sentence otherwise.

---

If it is long, I would put it in italics.
Will add quotes.